### ISSUE TITLE
'Representing biologists' view of biology (integral to membrane)

### ISSUE THREAD

**Issue by RLovering** [geneontology/go-annotation#1621]
Hi All

This issue was discussed in November 2016 at GOC meeting described in the minutes:
Ruth - the need to combine evidence types
Examples: Plasma membrane & integral to plasma membrane: require 2 bits of evidence: localization + sequence
Presentation on agenda discussed. 
5 possible ways forward
1. Allowing IDA for the full ‘integral to synaptic vesicle membrane’ term.
2. Extending IC annotations to allow IEA/NAS/TAS-supported annotations in the ‘with’ field. 
3. Requesting and using a new ECO code (possibly under ‘combinatorial evidence ; ECO_0000212’). 
4. Use LEGO annotation
5. Create these annotations automatically, e.g. if the protein is annotated to ‘SV’ and ‘integral to membrane’
**Decision: everyone in the room (apart from 3 people) agreed that option 1, using IDA evidence code to support the application of the ‘integral to synaptic vesicle membrane’ term, based on (e.g.) immunofluorescence demonstrating a protein is located in the ‘synaptic vesicle membrane’, and InterPro annotation confirming the protein is ‘integral to membrane’; i.e. a combination of IDA and IEA evidence.**
However, it was noted that in future multiple evidence codes could be included in the annotation, in which case this decision should be reviewed. 
Jim pointed out that if each annotation had a unique number then it would be possible combine specific annotations to create the combination annotation.

A powerpoint presentation was included with this item [http://wiki.geneontology.org/index.php/File:RFOULGER_IC_membrane_Sept_2016.pptx] Please look at this before responding to this ticket so that we don't have to restate what is listed in the presentation.

Key issues to consider:

1. If we chose a new evidence code for IDA (IMP etc) **plus** ISS (for now abbreviated to IDA-ISS) will these annotations get transferred by PAINT and Compara to orthogous genes. 
2. Is anyone willing to look at the evidence for 50 mammalian annotations in the next 3 weeks to determine which of these rely solely on the experimental data and do not use any prior knowledge? Thus get a sense of the scale of this issue.
3. I suspect >90% IDA will be IDA-ISS (IMP-ISS etc), certainly for human at least. Will all of these be updated to IDA-ISS en masse? Or would it be more sensible to create a more specific code eg: DDA (no longer inferred! but demonstrated by a direct assay).

Ruth

@dosumis @ukemi @krchristie @vanaukenk @ValWood @selewis @thomaspd @pgaudet @rachhuntley @srengel @mah11 @hattrill please could someone tag Judy as I don't know her @ account.

**Comment by srengel** [comment]
tagging Judy @judyblake

**Comment by mah11** [comment]
sorry, just remembered that I was flagged on this one

I oppose using plain IDA (option 1) in the situation described, because it would be misleading. In the example,

- a fluorescence microscopy experiment provides direct-assay evidence that a protein localizes to the synaptic vesicle; and
- a sequence match indicates that the protein is integral to some membrane.

Although biologists do want us to put the two pieces of information together, and infer that the protein is integral to the synaptic vesicle membrane, it does not follow that we should attribute the inference to only one of the two lines of evidence it requires. Biologists understand that fluorescence microscopy cannot show that any protein is integral to any membrane, and they do not want us to claim that it does.

Of the remaining options, 2 and 3 sound the most viable, and I don't have a very strong preference between them. I admit I don't understand how GO-CAM (fka LEGO; option 4) would solve the problem. Option 5 looks orthogonal to the evidence question - whether inferences are manual or automated, they would still require accurate evidence.

**Comment by ValWood** [comment]
For these we always  do

integral to plasma membrane IC from GO:0005886|GO:0016021

where
GO:0005886    plasma membrane  IDA
GO:0016021    integral component of membrane ISS

Its simple, it works with existing mechanisms and it doesn't fudge evidence codes.
You are making a curator inference based on 2 separate pieces of evidence.
Isn't this even a documented use case for IC?

(then we hide GO:0016021    integral component of membrane ISS in the summary annotation view)

**Comment by krchristie** [comment]
> 1. Allowing IDA for the full ‘integral to synaptic vesicle membrane’ term.

I strongly oppose option 1 to just use plain IDA, for exactly the same reasons Midori does. I am very much against fudging evidence codes. When I started in GO in 2000, one of the strengths of GO that people were pointing out was that it was accurately tracing the evidence so that you could figure out why something was said, unlike the situation that existed then where many sequence records were labeled with things like "putative RNA helicase" and there was no way to figure out where this came from. GO should remain true to its principal and provide clarity of evidence.

> 2. Extending IC annotations to allow IEA/NAS/TAS-supported annotations in the ‘with’ field.

I am OK with option 2 to extend IC, at least to allowing IEA to cover things like InterPro domains. I am not keen on allowing NAS or TAS as these include some old annotations that are truly garbage. However, it seems that there are several people who are not satisfied with this solution. Also, IC annotations don't show up in PAINT and thus cannot be the source of phylogenetic propagations via the PAINT tool.

> 3. Requesting and using a new ECO code (possibly under ‘combinatorial evidence ; ECO_0000212’).

In concept, I like the idea of option 3 to make combinatorial evidence codes to more clearly indicate that the statement is a combination of experimental. I am fine with the fact that it seems we might need several flavors of this: IDA-ISM, IDA, ISS, IDA-IEA (and so forth for other experimental evidence codes). 

If we allow these combinatorial, part experimental & part sequence codes, I think we should allow them for propagation via PAINT, at least if they have been manually assigned.

> 4. Use LEGO annotation

I do not think think that GO-CAM annotation is a complete solution to this issue. While the GO-CAM will allow putting different evidence on different statements, and will be fine when someone is viewing the model, we don't currently have any way to display GO-CAMs to users. 

At the present time, and likely for some time to come, we will be dependent on GPAD exports from GO-CAMs in order to populate the GO annotations that can be displayed on MOD pages for genes. For this, I think that combinatorial evidence (especially option 3, though option 2 would also be feasible) may still be desirable when exporting the annotations to these "integral to membrane" GO terms and also to other GO terms that require multiple lines of evidence.

> 5. Create these annotations automatically, e.g. if the protein is annotated to ‘SV’ and ‘integral to membrane’

This doesn't seem to be a fleshed out proposal. If we were to do this using new combinatorial evidence codes proposed in #3, then we would need to multiply those new codes by 2 and generate both the **manually annotated** and the **automatically inferred** versions, so that people and tools such as PAINT can distinguish between ones that curators made and ones that were made computationally.

**Comment by krchristie** [comment]
> I suspect >90% IDA will be IDA-ISS (IMP-ISS etc), certainly for human at least. Will all of these be updated to IDA-ISS en masse? Or would it be more sensible to create a more specific code eg: DDA (no longer inferred! but demonstrated by a direct assay).

A. I think it would be a really bad idea to create a code DDA as distinct from IDA, especially if DDA is suppose to be the more specific code that would mean what we've used IDA to mean since the beginning, and IDA would now be allowed to be more general. I also think the difference between "demonstrated" and "inferred" is too subtle which would mean confusion about the difference between the two and lead to inconsistent usage.

I also still think that allowing a single experimental code to be used as the evidence for a GO term when it only demonstrates/infers a portion of what that GO term means suffers from the same objections that @mah11 and I have made to just using IDA (aka option 1).

B. Regarding transfer en masse from IDA to IDA-ISS, it seems to me that this would be a decision for each group to make based on their historical annotation practices and thoughts about what makes most sense for their annotations.

**Comment by ValWood** [comment]
But if we change these to IDA-ISS or whatever en masse, we lose what the  IDA experiment actually showed. IDA to plasma membrane/ER membrane etc.

I think this is an added confusion for uses, and unecessary work for everyone.

What is wrong with the current method of

integral to plasma membrane IC from GO:0005886|GO:0016021
where
GO:0005886 plasma membrane IDA
GO:0016021 integral component of membrane ISS

exactly?
it's very clear how the annotation was derived, and it reflects the experimental results.

**Comment by bmeldal** [comment]
We have a similar problem when annotating complexes: Often the function or location is only shown for one complex member (maybe the cataytic subunit) or part of the complex so we can't use IDA. We have created the complex-equivalent to IC, ECO:0005547 biological system reconstruction evidence based on inference from background scientific knowledge used in manual assertion. It's very conservative but save. Happy to get involve in the discussion of new, combinatorial ECO code(s) that would also capture our use cases.

See you next week!

**Comment by hattrill** [comment]
Somewhat tangental, but relevant to the discussion. I was interested whether biologists cared about the distinction between “integral to plasma member” and “plasma membrane”. I asked someone who works on plasma membrane profiling and asked him. And his remark surprised me:

“Personally, I think the following:
- I don't find the 'integral component of plasma membrane' or 'integral to plasma membrane' useful
- Likewise currently for 'plasma membrane' versus 'cell surface' since the annotation currently doesn't really distinguish anything. Only a small subset of plasma membrane proteins that are present on the cell surface are annotated as such. However, if the 'cell surface' term were expanded (and used in conjunction with 'plasma membrane') it would be useful - to reflect proteins that are actually externally facing rather than PM proteins that have no / minimal ectodomain - to me this is the key distinction.”

I explained that it wasn’t really within the scope of the GO. But I offered him a child term:

GO:0044214 spanning component of plasma membrane

And he said “I suppose GO:0044214 would help distinguish proteins that are likely to have an ectodomain (spanning) from those that may not - so this could be helpful.”

I think that when we use “integral to..” we often mean “spanning….” but this term only has 22 exp annotations, perhaps because it's horrible name or because we tend to drift towards an MFT (my favourite term).

@vanaukenk and I discussed this on a call (when we should have probably been talking about wnt signaling) and thought that we should consider changing the term name from ‘spanning component of plasma membrane’ -> ‘transmembrane component of plasma membrane’ to help people find and use it.

If you folks think that would be useful, I will make a ticket to change the name.

**Comment by dosumis** [comment]
> Only a small subset of plasma membrane proteins that are present on the cell surface are annotated as such. However, if the 'cell surface' term were expanded (and used in conjunction with 'plasma membrane') it would be useful - to reflect proteins that are actually externally facing rather than PM proteins that have no / minimal ectodomain - to me this is the key distinction.”

IIRC 'cell surface' needs to be wired up properly

> I think that when we use “integral to..” we often mean “spanning….” but this term only has 22 exp annotations, perhaps because it's horrible name or because we tend to drift towards an MFT (my favourite term).

Re name: agree horrible.  Should change to transmembrane.

The term has been around for some time, but until relatively recently was hidden away - not connected to the rest of the hierarchy.  It's also not one of the standard set of components generated for most membranes.  Easy to add though as we have a standard design pattern for these.

If it's reasonable to use a combination of presence of TM domain + location to membrane to infer 'transmembrane', perhaps this should be done in bulk from direct or indirect annotation to (X) membrane - scripting from some reliable source for domains.  Could be coupled with instantiating transmembrane terms for membranes generally.,

**Comment by ValWood** [comment]
I think the issue that myself @krchristie @mah11  have here is that IDA is way to strong an evidence code for these experiements because the precise location we would be annotating, well has not been assayed, so this is misleading.

I completely believe we should show what the experimenter is trying to demonstrate  too,  so I'm completely happy in most other cases to say (if the phenotype is conclusive), that a phenotype  supports  a process, or even an activity if its the right type of experiment.

In the situation here, I think we can infer the precise location from the experiment (taken together with other knowledge), but it isn't a direct assay. So why don't we just use the less loaded evidence code "EXP" (now I think about it I have plumped for this code quite a lot for an inference from a combination of experimental data (which might sometimes be supplemented by sequence information too). 

This reserved IDA for the situation when something *has* been precisely localized to a specific location by an assay as intended.

Would that work?

**Comment by hattrill** [comment]
Personally, I like Val's suggestion a lot.
I think that fits with the boarderline examples of the type discussed here which are usually based on >1 observation, a pinch of author intent and curator confidence rather than fig 2 unequivocally shows this.

**Comment by pgaudet** [comment]
Hello, 

I agree with the  suggestion from @ValWood,  @mah11 and @krchristie to annotate what we have evidence for). For me the problem that is not resolved is, which GO term does a FACS analysis provide evidence for. I would be happy if people annotated to 'plasma membrane for animal cells. I suppose that for cells with cell walls, if you do a FACS without removing the cell wall, then you are looking at the cell wall (right @tberardini ?) 

In PAINT cell wall and plasma membrane could be grouped by the parent 'cell periphery', which is also neutral in terms of sub-localization (external/internal/transmembrane, etc). 

Would that work ? 

Pascale

**Comment by pgaudet** [comment]
And WRT 'cell surface', I dont think this term is necessary. Annotations could probably be moved to 'plasma membrane' for animal-type cells. I am not sure what to suggest for other organisms, but it should be possible to use 
-cell periphery
-- cell wall 
-- plasma membrane 

to represent the general localization of proteins around the surface of any cell. 

---
@hattrill I had the same feedback from GO users that 'integral to the membrane' and all the other terms are not very useful. The other major issue is the huge inconsistency in usage (that I noticed multiple times while doing PAINT); these terms are really used as readouts; very few experiments provide evidence at a level greater than the three terms above (cell periphery, cell wall , plasma membrane). It might be worth reviewing usage of these terms on an annotation call.  

Pascale

Thanks, Pascale

**Comment by ValWood** [comment]
I think cell surface is very useful :)

cell periphery includes cell cortex, plasma membrane, cell wall (or external encapsulating structure)

cell surface includes
The external part of the cell wall and/or plasma membrane.
This can include things anchored intrinsically or  extrinsically to the cell wall or the plasma membrane.
Excreted digestive enzymes which we don't know if they are associated with the PM or the cell wall may be annotated directly to this term.

It is very useful to be able to identify all cell  surface molecules (if used correctly).

I  like "integral to membrane" too,  it is useful to distinguish "integral" from "anchored" etc but I would be less concerned if these were somewhat consolidated.

**Comment by ValWood** [comment]
Re FACs, I have never curated an experiment using FACs  for "cell surface" or "cell periphery" localization (we only appear to use for cell size sorting generally?)
@Antonialock  @mah11  have you ever?

**Comment by ValWood** [comment]
Re cell surface/ cell periphery  the thing we are using it to distinguish is whether a gene product is inside or outside the cell. This  is useful, especially if we do not know much else... and for many unknowns we do at least know this (or can infer with high confidence).

**Comment by mah11** [comment]
I have not used FACS for localization (CC) annotations; it seems to be more of a thing for cells derived from multicellular organisms.

**Comment by ValWood** [comment]
Re cell surface, so we now have a nice list of the things we know are *outside* the cell
https://www.pombase.org/term/GO:0009986
(and a TPV for cleavage furrow!)

**Comment by ValWood** [comment]
(and a TPV for cleavage furrow!)
or an annotation error, oops...
https://github.com/pombase/curation/issues/1846

**Comment by addiehl** [comment]
As a matter of record, back when I worked as a GO annotator, I routinely annotated flow cytometry data about the surface expression of proteins to 'external side of plasma membrane', the reasoning being that the protein in question was clearly exposed on the external side.  This annotation provides no information as to how the protein is connected to the cell, simply that a portion of it is at the external side of the plasma membrane.  Such annotations are compatible with the definition of 'external side of plasma membrane':  "The leaflet of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface."

Obviously, many cell surface proteins are transmembrane, and have cytoplasmic portions as well, and other types of assays can provide data to support annotations that provide additional details about the localization of such proteins.

-- Alex

**Comment by pgaudet** [comment]
Thanks Alex, that's useful to know. Looks like the term is being used as a read-out rather than as the known location. We really need guidelines ! 

Pascale

**Comment by pgaudet** [comment]
Hi @krchristie 
I am curious to know what structures are external to the plasma membrane, the ECM ? 


Thanks, Pascale

**Comment by krchristie** [comment]
Yes, ECM is external to plasma membrane, though not sure ECM would come through in FACS. 

However, I remain uncomfortable trying to say 'plasma membrane' based on a FACS experiment. Some of these transmembrane proteins are cleaved as part of their normal functioning and the FACS experiment doesn't indicate whether the entity identified contained a transmembrane section, or was only a fragment that is peripheral to the membrane.

**Comment by pgaudet** [comment]
OK - for me 
- GO:0009897 external side of plasma membrane
- GO:0009898 cytoplasmic side of plasma membrane
should be disjoint from membrane spanning

So if you know that a protein is membrane spanning, annotating to 'external side of plasma membrane' seems misleading. Again I'd be in favor of using simply 'plasma membrane' for cases where you know from sequence that the protein is membrane spanning. This way you are not making an annotation for something you don't have evidence for (which would be 'membrane spanning), but you are not stating that most of the protein is located externally (which is what 'external side of plasma membrane' implies IMO). 

Thanks, Pascale

**Comment by pgaudet** [comment]
We probably need a decision tree ! ;)

**Comment by RLovering** [comment]
I like Pascale's suggestion: WRT 'cell surface', I dont think this term is necessary. Annotations could probably be moved to 'plasma membrane' for animal-type cells. I am not sure what to suggest for other organisms, but it should be possible to use
-cell periphery
-- cell wall
-- plasma membrane

The definition for cell periphery is:
The part of a cell encompassing the cell cortex, the plasma membrane, and any external encapsulating structures.
So at the very least 'cell surface' should be a child of 'cell periphery'

then use GO:0019897 extrinsic component of plasma membrane (or equivalent cell wall term)
Definition The component of a plasma membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region. 

If there is no attachment to the plasma membrane or cell wall then the protein should be annotated with extracellular region.

**Comment by RLovering** [comment]
Looking at the child terms for plasma membrane (especially GO:0019897 extrinsic component of plasma membrane) FACs analysis of animal cells would be annotated with plasma membrane rather cell surface with IC or IDA depending on the outcome of this discussion above to the more descriptive GO terms based on the protein structure.

**Comment by RLovering** [comment]
To respond to the comments about 'Representing biologists' view of biology (integral to membrane)
I have tried to put all comments together because it is really hard to follow this now, apologies if I have missed anything. From what I can see the discussion ended with a suggestion by Val to use EXP when including ISS/IEA/background knowledge with an experimental annotations

Firstly can I correct this statement:
integral to plasma membrane IC from GO:0005886|GO:0016021
where
GO:0005886 plasma membrane IDA
GO:0016021 integral component of membrane ISS
Often the GO:0016021 integral component of membrane is actually an IEA from InterPro not an ISS. Currently IC cannot be supported with IEA annotations.

Val's suggestion : wrt integral to plasma membrane based on IDA and ISS/IEA: I think we can infer the precise location from the experiment (taken together with other knowledge), but it isn't a direct assay. So why don't we just use the less loaded evidence code "EXP". This reserves IDA for the situation when something has been precisely localized to a specific location by an assay as intended.
> this would mean redefining IDA (and IMP and others?). I don't know if EXP annotation can be propagated to orthologs. This would sort out this problem but I think  we have to accept that would then get used by curators for BP and MF annotations too.

my comments to this discussion (which I read from the top so may be unnecessary if we are going with Val's EXP suggestion)
1. Many biologists just want an annotation dataset that they can use. For example it is very difficult to track the experimental evidence to support Reactome annotations, and yet this is becoming a more popular resource in functional analysis than GO. But this also comes down to a point I have made before which is that GO as many different user communities. 
2. We provide PMIDs so that people can check the evidence supporting the annotations themselves. People using GO on a gene by gene basis should do this. People using GO for large lists of genes will not do this and actually will not care if 0.1% of the membrane GO annotations have been made to represent biologists view and are only 'inferred' from the direct experimental evidence.
3. There is nothing to stop individual MODs applying a stricter application of evidence codes than the GOC requires. for example some MODs might decide that when it applies protein structure information it will ALWAYS use IC then that would not be a conflict with the rest of the GOC deciding that 'inferred by direct assay' can include 'author intent' or 'known protein structure' information. However, and I have said this a few times before, annotation with the IC evidence code will not be transferred to orthologous proteins in other species. Therefore, if MODs are supported on grants based on the medical relevance of their data, they might want to consider how they are going to get their annotations propagated to medically relevant organisms (ie human). 
4. Another option could be to allow IC data to be propagated to orthologs. This could be done by lifting this restriction for PAINT and Compara.
5. I object to the suggestion that IDA can be used for author intent when annotating BP and MF (which seems to be the implication here) but not used for CC. How do you justify this in guidelines, to curators, to users, to SAB? I think that once you make this decision then we are going to spend more time deciding when to use IC or IDA for every experiment and we will create fewer and fewer annotations.

An example annotation:
P08913 ADRA2A Alpha-2A adrenergic receptor PMID: 10948191 An asn to lys polymorphism in the third intracellular loop of the human alpha 2A-adrenergic receptor imparts enhanced agonist-promoted Gi coupling. Annotation: integral component of plasma membrane. This annotation is actually based on: Expression, ligand binding, and basal receptor function. Based on my knowledge of receptors and author intent behind their experiment of transfecting ADRA2A into CHO and COS-7 cells. I think IDA is an OK evidence here, the alternative is to use IC WITH ligand binding; alpha2-adrenergic receptor activity. Expression was determined by ligand binding.

Other points raised:
6. Biologists understand that fluorescence microscopy cannot show that any protein is integral to any membrane, and they do not want us to claim that it does.
Where is the data to support this claim? What we do know from one scientist is: "it would be useful to reflect proteins that are actually externally facing rather than PM proteins that have no / minimal ectodomain - to me this is the key distinction.”

7. 'When I started in GO in 2000, one of the strengths of GO that people were pointing out was that it was accurately tracing the evidence' this is simply not true. It may have been true at SGD but the evidence code has always included 'inferred' and for many groups we have been encouraged to only annotate to our own level of understanding of the data but to consider background knowledge. 

8. the implication of using a new ECO code (possibly under ‘combinatorial evidence ; ECO_0000212’ is that many of the GPAD exports from GO-CAMs used to populate the GO annotation files are likely to all end up using combinatorial evidence codes. The use of very specific evidence codes has a lot of positive aspects to it, however, it will reduce considerably annotation output.

9.  IDA v IDA-ISS, IDA-IEA v DDA (demonstrated by direct assay) I would prefer DDA as a new term to be added when data supports it, as I think the majority of human data is inferred (eg reporter assays, westerns of subcellular compartments (assuming good purification) but I can see that for other MODs they might have much of their data confirmed by clean direct experimental evidence. I think if there is going to be a new code then the GOC will need to decide how IDA is defined.

 10: please rename GO:0044214 spanning component of plasma membrane to transmembrane component of plasma membrane

11. from David OS: If it's reasonable to use a combination of presence of TM domain + location to membrane to infer 'transmembrane', perhaps this should be done in bulk from direct or indirect annotation to (X) membrane - scripting from some reliable source for domains. Could be coupled with instantiating transmembrane terms for membranes generally.

12. fromDavid OS:  'cell surface' needs to be wired up properly (or removed see Pascale comment above)

**Comment by ValWood** [comment]
RL > I object to the suggestion that IDA can be used for author intent when annotating BP and MF (which seems to be the implication here) but not used for CC. How do you justify this in guidelines, to curators, to users, to SAB? I think that once you make this decision then we are going to spend more time deciding when to use IC or IDA for every experiment and we will create fewer and fewer annotations.

I would explain this by curator judgement -when is there enough information  presented in the experiment to make this assertion using a particular evidence code. In this case I just feel that there isn't  enough information from plasma membrane location and the presence of TMM to (in most cases) to make this a general rule.  

Although, there are other ways to get this annotation by IDA, 
for example if you know that a GP is a member of a specific complex that is defined to be part of the  GO:0097648 https://www.ebi.ac.uk/QuickGO/term/GO:0097648

Although as  you say
> There is nothing to stop individual MODs applying a stricter application of evidence codes than the GOC requires. 

 I won't object if others want to implement less strict criteria. but we will continue to use curator judgement and not make "integral to membrane"  IDA from flourescence microscopy. I don't think our users  would approve.

**Comment by krchristie** [comment]
I do not think that I agree with this:

>    GO:0009897 external side of plasma membrane
>     GO:0009898 cytoplasmic side of plasma membrane
>     should be disjoint from membrane spanning

I think that there are quite a lot of membrane spanning proteins that have domains that are accessible on the cell surface. This seems like a requirement for function for many receptors that need to be accessible on the cell surface and also capable of transmitting a signal to the interior of the cell. 

Often the main point of a FACS analysis is to determine if a given protein is accessible on the cell surface. Is this true for all transmembrane proteins? It seems like it might not be depending on the size and possible modifications of the external domain. I don't see why it is a problem to say that something is transmembrane and also cell surface, when both are true.

**Comment by bmeldal** [comment]
GO:0089717 spanning component of membrane
Def: The component of a membrane consisting of gene products and protein complexes that have some part that spans both leaflets of the membrane.

It says nothing about whether the protein or complex also have a extracellular or cytoplasmic domain. I agree with Karen that one would likely want to annotate to both if the data was available. (I'm not going to get involved in the IDA debate!)

On the flip side, the ecoli Maltose transport complex 
https://www.ebi.ac.uk/complexportal/complex/EBI-6477643
spans the inner plasma membrane and has two transmembrane components (malF & malG) that have neither periplasmic nor cytoplasmic domains.

Birgit

**Comment by pgaudet** [comment]
Notes from GO meeting NYC 2018 breakout group: 
Annotation of membrane proteins
Do we need to have membrane topology CC terms in GO?
Agreed that all integral to membrane IEA annotations Plus plasma (other specific organelle) membrane IDA annotations will be changed to integral to plasma (other specific organelle) membrane IDA by automated pipeline.
Agreed: Guidelines that evidence such as immunofluorescence can be used to create a integral to ‘specific’ membrane if the protein already has ‘integral to membrane’ supported by IEA (e.g., InterPro) evidence, in this example the evidence IDA would be applied.
Agreed: Guidelines will suggest that if more granular evidence is available then use a more specific evidence code than IDA - pending checking ECO hierarchy.
Pascale to look at obsoleting GO membrane terms with very few annotations. Here, we also need to consider the implications for obsoleting these terms wrt modeling, e.g. transmembrane signaling receptors and their sub-functions, in GO-CAM.

**Comment by RLovering** [comment]
looks good to me
